substrate/bin/utils/chain-spec-builder/Cargo.toml

```diff
@@ -21,15 +21,28 @@ path = "bin/main.rs"
 name = "chain-spec-builder"
 
 [lib]
-crate-type = ["rlib"]
+# Docs tests are not needed since the code samples that would be executed
```

Comment:
I am not sure why this is needed; anything in `docify::embed` already has a `ignore` (for the opposite, you have `docify::embed_run!`).

Reply:
It is about compiling test snippet not running it. 

`docify::embed` still compiles the snippet, and since we use the trick with `bash!` macro (which is inaccessible in doc context), compilation would fail.

Reply:
Adding on top of what @michalkucharczyk said, docify::embed in context of markdown will use `rust` lang instead of `ignore`, which will make the sample a doc test. It might insert `ignore` accordingly for general public/crate-level docs from `.rs` files (which I haven't tested), so this can be considered a bug. Also, embedding in markdown files can not use `embed_run`. I tried that previously.